File: ProgrammersNET/ProgrammersNET.Tests/AmountConvertTest.cs
using Microsoft.VisualStudio.TestTools.UnitTesting;
using System;
using ProgrammersNET;

namespace ProgrammersNET.Tests
{
    [TestClass]
    public class AmountConvertTest
    {
        [TestMethod]
        public void AmountConvert_ShouldReturnTrue_For2500()
        {
            string stringResult = AmountConvertPage.ConvertAmountToWords(2500);
            Assert.AreEqual(stringResult, "Two Thousand Five Hundred and 00/100 dollars");
        }       
    }
}

File: ProgrammersNET/ProjectVB.Tests/AmountConvertTest.vb
Imports System.Web
Imports System.Text
Imports Microsoft.VisualStudio.TestTools.UnitTesting
Imports ProjectVB

<TestClass()> Public Class AmountConvertTest

    <TestMethod()> Public Sub AmountConvert_ShouldReturnTrue_For2500()
        Assert.AreEqual(AmountConvertPage.ConvertAmountToWords(2500), "Two Thousand Five Hundred and 00/100 dollars")
    End Sub
End Class